lib/tasks/sequence.rake

```diff
@@ -0,0 +1,18 @@
+# TRANSLATORS: do not translate
+desc <<-END_DESC
+  Useful tasks for SQL sequences.
```

Comment:
```suggestion
  Reset PostgreSQL sequences to their max values. Does nothing on other databases.
```

Reply:
Well it's explained on the second line here, this is rather generic sentence for the whole namespace but I am okay with this. Rebased.